src/drawers/areaDrawer.ts

```diff
@@ -11,9 +11,15 @@ export module Drawers {
       this._areaSelection.datum(data);
     }
 
-    public setup(area: d3.Selection<void>) {
-      Drawer.prototype.setup.call(this, area);
+    public renderArea(): d3.Selection<void>;
+    public renderArea(area: d3.Selection<void>): Drawer;
+    public renderArea(area?: d3.Selection<void>): any {
+      if (area == null) {
+        return super.renderArea();
+      }
+      Drawer.prototype.renderArea.call(this, area);
```

Comment:
Calling `super()` would attach a line since `Drawers.Area` extends `Drawers.Line`, so I assume the `.call()` structure was added on purpose.

This weirdness suggests that something is off about the inheritance (which switching to `Drawers.Path` would probably fix...).